packages/blob-storage-manager/test/storages/SwarmStorage.test.ts

```diff
@@ -5,36 +5,32 @@ import { testValidError } from "@blobscan/test";
 import { env } from "../../src";
 import { SwarmStorageMock as SwarmStorage } from "../../src/__mocks__/SwarmStorage";
 import { BlobStorageError } from "../../src/errors";
-import {
-  BLOB_DATA,
-  BLOB_HASH,
-  SWARM_REFERENCE,
-  SWARM_STORAGE_CONFIG,
-} from "../fixtures";
+import { BLOB_DATA, BLOB_HASH, SWARM_REFERENCE } from "../fixtures";
 
 describe("SwarmStorage", () => {
   let storage: SwarmStorage;
 
   beforeAll(() => {
-    storage = new SwarmStorage(SWARM_STORAGE_CONFIG);
+    storage = new SwarmStorage({
+      chainId: env.CHAIN_ID,
+      beeEndpoint: env.BEE_ENDPOINT ?? "",
```

Comment:
Since these are tests, we can safely assume that the variable is set, right?

```suggestion
      beeEndpoint: env.BEE_ENDPOINT,
```

Reply:
The environment variable is defined as optional, so TypeScript will treat it as such, even if it's defined in the `env.test` file.

I'll ensure it's not undefined by throwing an error at the beginning of the test file.